boards/shields/mikroe_adc_click/doc/index.rst

```diff
@@ -35,8 +35,8 @@ Set ``-DSHIELD=mikro_adc_click`` when you invoke ``west build``. For
 example:
 
 .. zephyr-app-commands::
-   :zephyr-app: test/boards/board_shell
-   :board: frdm_k64f
+   :zephyr-app: samples/hello_world
+   :board: lpcxpresso55s16
```

Comment:
`:gen-args: -DCONFIG_SHELL=y` ?

Reply:
Why?

Reply:
Well, otherwise you commit message not seem very clever to me, why else would you refer to https://github.com/zephyrproject-rtos/zephyr/commit/7c85f4b2f5bea25a3c34a8f81ef6590639629434 or board_shell test mentioned here in the first place? And, why not samples/drivers/adc?

Reply:
I have searched twice for a reasonable explanation and have not found one.

Reply:
The commit is replacing the dead (double dead actually, as it had a typo in "tests/..." in the first place) link to a sample that doesn't exist anymore, with a link to hello world which is much more likely to just work (...and shouldn't require CONFIG_SHELL?)

Reply:
How is the ADC shield supposed to work with the hello_world sample, for what is it there? If this shield does not work at all, then this PR should not be merged yet, broken is broken. But as I understand it, it does not work for a specific board.

Reply:
Thank you for the constructive feedback.